app/code/Magento/InventoryInStorePickupApi/etc/webapi.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
```

Comment:
Some API-functional tests would be extremely helpful here.

Reply:
Issue has been created #2264.